backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Proprietary.java

```diff
@@ -225,6 +225,21 @@ public Optional<Gegenstandlos> getGegenstandlos() {
     );
   }
 
+  /**
+   * Set the gegenstandlos metadata element (if it does not exist, it will create it).
+   * @param date - date to set as gegenstandslos with
+   */
+  public void setGegenstandlos(final String date) {
```

Comment:
I'm fine with having this method here but I think I would have added a `Gegenstandlos getOrCreateGegenstandlos()` method here, called that from the `Norm` class and then set the date on the `Gegenstandlos` instance directly